src/styles/blocks/photo.scss

```diff
@@ -0,0 +1,10 @@
+.card__photo {
+  padding-bottom: 40px;
+  display: flex;
+  justify-content: center;
+
+  &-container {
+    height: 134px;
```

Comment:
Move to the `card` file and use nesting as it's related to it and it's not a separate BEM block